main.go

```diff
@@ -160,14 +161,41 @@ func main() {
 		fmt.Printf("Current task description: \n%+v \n", taskDesc)
 	}
 
-	containerDef := taskDesc.TaskDefinition.ContainerDefinitions[0]
-	oldImage := containerDef.Image
-	{
-		x := *targetImage
-		if *targetImage == "" {
-			x = fmt.Sprintf("%s:%s", *repoName, *sha)
+	var containerDef *ecs.ContainerDefinition
+	var oldImage *string
+	var i int
+	// multiContainer service
+	if *multiContainer {
+		containerDef = taskDesc.TaskDefinition.ContainerDefinitions[0]
+		oldImage = containerDef.Image
+		{
+			x := *targetImage
+			if *targetImage == "" {
+				x = fmt.Sprintf("%s:%s", *repoName, *sha)
+			}
+			containerDef.Image = &x
+		}
+	} else {
+		fmt.Printf("Task definition has multiple containers \n")
```

Comment:
Is the check above inverted? If we got here, then `*multiContainer == false`, which seems like the opposite of what I'd expect.

Reply:
duh.. it is my bad.